apex-benchmarks/src/main/java/apex/benchmark/ApplicationDimensionComputation.java

```diff
@@ -106,28 +112,52 @@ public void populateDimensionsDAG(DAG dag, Configuration conf, DefaultOutputPort
     dimensions.setConfigurationSchemaJSON(eventSchema);
     dimensions.setUnifier(new DimensionsComputationUnifierImpl<InputEvent, Aggregate>());
 
-    dag.setUnifierAttribute(dimensions.output, OperatorContext.MEMORY_MB, 4096);
-    dag.setAttribute(dimensions, Context.OperatorContext.APPLICATION_WINDOW_COUNT, 10);
+    dag.setUnifierAttribute(dimensions.output, OperatorContext.MEMORY_MB, 10240);
+//    dag.setAttribute(dimensions, Context.OperatorContext.APPLICATION_WINDOW_COUNT, 120);
+//    dag.setAttribute(dimensions, Context.OperatorContext.CHECKPOINT_WINDOW_COUNT, 120);
+    
     dag.setInputPortAttribute(dimensions.input, Context.PortContext.PARTITION_PARALLEL, true);
     
     // store
     AppDataSingleSchemaDimensionStoreHDHT store = createStore(dag, conf, eventSchema); 
+    store.setCacheWindowDuration(10000 * 5 / STREAMING_WINDOW_SIZE_MILLIS);   //cache for 5 windows
```

Comment:
In the application, the application window in fact same as stream window. The previous application window meant the window of 10 second